src/components/grid-aware/StatsBlock/StatsBlock.module.css

```
@@ -0,0 +1,92 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding-bottom: 150px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 38px;
+    padding-top: 50px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column-start: 2;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  grid-column-end: 11;
+  grid-column-start: 2;
+  margin-bottom: 54px;
+}
+
+.statCard {
+  margin-right: 85px;
+}
+
+.number {
+  font: var(--font-family-lato);
+  font-size: 72px;
+  margin-bottom: 5px;
+  width: 250px;
+}
+
+.statement {
+  font: var(--font-body-medium);
+}
+
+@media (--tablet-and-down) {
+  .statement {
+    margin-bottom: 40px;
+  }
+
+  .number {
+    font-weight: bold;
+    line-height: 65px;
```

Two things:

1. You need to shrink the font size on mobile to 50px.
2. You should generally use a [unitless value for line-height](https://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/), because otherwise you may run into issues when nesting styles. 65/50 = 1.3, so that's the value you should use for line height.

```suggestion
    font-size: 50px;
    line-height: 1.3;
```